addons/l10n_eg/data/account_tax_group_data.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <!-- Account Tax Group -->
+        <record id="eg_tax_group_other" model="account.tax.group">
+            <field name="name">Other Taxes</field>
+        </record>
+
+        <record id="eg_tax_group_stamp" model="account.tax.group">
+            <field name="name">Stamp Tax</field>
+        </record>
+
+        <record id="eg_tax_group_withholding" model="account.tax.group">
+            <field name="name">Withholding Tax</field>
```

Comment:
Since v15, we have the preceding_subtotal field on the tax group, which makes it possible to have a subtotal without the withholding.  Is Egypt used to the withholding amounts being subtracted immediately or is it good to indicate the amount before subtracting the withholding?

Reply:
we need to indicate the amount before subtracting.

Reply:
@akr-odoo Please try using it and see if it does what you need.